### PR TITLE
fix(cluster.py): missing normilize_ipv6_url from

### PR DESCRIPTION
**Pull request by fgelcer**
_register_grafana_json function.
found it using latest curl version, as it doesn't
accept square brackets.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -974,7 +974,8 @@ def get_backtraces(self):
             if self.is_ubuntu14():
                 # fixme: ubuntu14 doesn't has coredumpctl, skip it.
                 return
-            core_pids = self._get_core_pids()
+            core_pids = None
+            # core_pids = self._get_core_pids()
```
Comment (juliayakovlev) [inline sdcm/cluster.py:978]: Why this line commented?

Comment (fgelcer) [reply sdcm/cluster.py]: oh, it got in this PR by mistake...
i commented it out because of #2217, but it shouldn't get in the code...

**Comment by fgelcer** [comment]
> Please, address the comment

done

**Comment by bentsi** [comment]
@fgelcer please use fstrings in your future PRs

**Comment by fgelcer** [comment]
> Are we sure this would work with the brackets ? It had escaping for a reason...
> 
> I think it might break on ipv6 address...

the function that runs this normalize_ipv6_url is in place for a while, and it seems to work (i didn't test in this specific case though)

**Comment by fruch** [comment]
> > Are we sure this would work with the brackets ? It had escaping for a reason...
> > I think it might break on ipv6 address...
> 
> the function that runs this normalize_ipv6_url is in place for a while, and it seems to work (i didn't test in this specific case though)

In this case is being use in a bash command which I didn't followed, but I believe the escaping was for a reason

**Pull request merged**